### PR TITLE
test: add tests for transformation slate to newsdoc

### PR DESCRIPTION
**Pull request by gularsson**

### CODE REVIEW

File: src-srv/utils/CollaborationServer.ts [hunk 1]
```diff
@@ -218,13 +218,14 @@ export class CollaborationServer {
...
     // Fetch content
-    const { document = null } = await this.#repository.getDoc({
+    const { document = null, ...rest } = await this.#repository.getDoc({
       uuid,
       accessToken: context.token
     }) || {}
 
 
     if (document) {
+      console.log(rest)
```
Comment (dannelundqvist) [inline src-srv/utils/CollaborationServer.ts:228]: Leftover that should be removed?

File: src-srv/utils/transformations/newsdoc/core/text.ts [hunk 2]
```diff
@@ -26,9 +26,10 @@ const replace = (type: string, translateList: Record<string, string>): string =>
 }
 
 const createDocument = (): Document => {
+  // TODO: does this work, we'll find outj
```
Comment (dannelundqvist) [inline src-srv/utils/transformations/newsdoc/core/text.ts:29]: It seems like it's working? Should the comment be removed?

**Review by dannelundqvist — APPROVED** [review]
Good tests! Two cleanliness questions.

File: src-srv/utils/transformations/newsdoc/core/text.ts (hunk 2 above)
Comment (gularsson) [reply src-srv/utils/transformations/newsdoc/core/text.ts:29]: Next attempt/PR is to revert and save. Then we'll find out

**Pull request merged**